SIT.Manager.Avalonia/ViewModels/ToolsPageViewModel.cs

```diff
@@ -217,4 +236,115 @@ private async Task ClearCache()
             }
         }
     }
+
+    private async Task GenerateDiagnosticReport()
+    {
+        var results = await new SelectLogsDialog().ShowAsync();
```

Comment:
Would it make more sense if everything after here was in some kind of service, especially if it's likely that this could be added to/extended in the future?